tensorflow/python/keras/backend_test.py

```diff
@@ -1736,5 +1736,34 @@ def test_batch_normalization(self):
     self.assertEqual(normed.shape.as_list(), [10, 3, 5, 5])
 
 
+class ControlOpsTests(test.TestCase):
+
+  def test_function_switch_basics(self):
+    pred = keras.backend.placeholder([], dtype='bool')
+
+    def false_fun():
+      return array_ops.constant(False)
+
+    with self.cached_session():
+      x2 = keras.backend.switch(pred, lambda: array_ops.constant(True), false_fun)
+      x1 = keras.backend.function(pred, x2)([True])
+      self.assertAllClose(True, x1)
+      x1 = keras.backend.function(pred, x2)([False])
+      self.assertAllClose(False, x1)
+
+  def test_function_basic_error(self):
+    x = ops.convert_to_tensor(np.array([[1, 2, 3], [4, 5, 6]]),
+                              dtype='float32')
+    y1 = ops.convert_to_tensor(np.array([1, 2, 3]), dtype='float32')
+    def true_fun():
```

Comment:
let rename this to true_func.

Reply:
Sure, same is updated in the code.

Regards
Amit